levels_test.go

```diff
@@ -1049,3 +1052,41 @@ func TestKeyVersions(t *testing.T) {
 		})
 	})
 }
+
+func TestTableContainsPrefix(t *testing.T) {
+	opts := table.Options{
+		BlockSize:          4 * 1024,
+		BloomFalsePositive: 0.01,
+	}
+
+	buildTable := func(keys []string) *table.Table {
+		filename := fmt.Sprintf("%s%s%d.sst", os.TempDir(), string(os.PathSeparator), rand.Uint32())
```

Comment:
Can we use `ioutil.TempFile()`?. It makes sure that it creates a non-conflicting file.

Reply:
That can be done. But that would require more handling of the rest of the logic like `CreateTable`, etc. For simplicity, keeping it the same.